hashicorp/boundary-ui
# refactor: 💡 Update manage dropdown for host-catalogs

✅ Closes: https://hashicorp.atlassian.net/browse/ICU-14664

# Description
Updates Manage dropdown in Host-Catalogs to HDS

## Screenshots (if appropriate)
**Before**
![host catalog before1](https://github.com/user-attachments/assets/01539859-71a7-4a81-bd84-b2a11e9b1cce)
![host catalog before2](https://github.com/user-attachments/assets/8f32ea87-ff4f-479c-9642-6020f427940f)

**After**
![host catalog after1](https://github.com/user-attachments/assets/0c1c574d-31c2-489c-a0d0-82c7423d36fc)
![host catalog after2](https://github.com/user-attachments/assets/69e7f9fa-fb15-48fd-a745-0bc30bea763a)


## How to Test
1. Start Boundary Client
2. Select org from sidebar and select an org in table.
3. Select project from sidebar and select an project in table.
4. Select Host catalog from sidebar and select a host catalog
5. HDS Dropdown should be reflected 

## Checklist

~- [ ] I have added JSON response output for API changes~
~- [ ] I have added steps to reproduce and test for bug fixes in the description~
~- [ ] I have commented on my code, particularly in hard-to-understand areas~
~- [ ] My changes generate no new warnings~
~- [ ] I have added tests that prove my fix is effective or that my feature works~
